backend/selectgen.ml

```diff
@@ -327,6 +339,10 @@ method is_simple_expr = function
   | Csequence(e1, e2) -> self#is_simple_expr e1 && self#is_simple_expr e2
   | Cop(op, args, _) ->
       begin match op with
+      (* inlined ops without effects and coeffects are simple
+         if their arguments are *)
+      | Cextcall { effects = No_effects; coeffects = No_coeffects; } ->
```

Comment:
I'm nervous about making this change until the `c_builtin` field is here.  Can we please revert these four lines?

Reply:
ah, the comment about "inlined ops" is no longer relevant, but this new case in `is_simple_expr` is no different from the new case below in `effects_of` method - are you nervous about both of them? Also, the first version this PR had a constraint on builtin=true here, but I thought that you said it is not needed. Should I put it back?
There is no harm in merging this change as is, because the effects and coeffects are set conservatively everywhere at the moment. Equally, we can hold off on this PR until we merge the one that more precise effects and coeffects. The existing situation is safe.

Reply:
If the comment's no longer relevant, can it be removed?

Reply:
I've rephrase the comment to talk about Cextcall (instead of the non-exiting inlined_ops), I think it's accurate and useful.